_po/ja/reference/modules/pgroonga-standby-maintainer.po

```diff
@@ -0,0 +1,151 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Language: ja\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+msgid ""
+"---\n"
+"title: pgroonga_standby_maintainer module\n"
+"upper_level: ../\n"
+"---"
+msgstr ""
+
+msgid "# `pgroonga_standby_maintainer` module"
+msgstr ""
+
+msgid "Since 2.4.2."
+msgstr "2.4.2で追加。"
+
+msgid "This is still an experimental feature."
+msgstr "まだ実験的な機能です。"
+
+msgid "## Summary"
+msgstr "## 概要"
+
+msgid ""
+"The `pgroonga_standby_maintainer` module provides executing [the `pgroonga_wal"
+"_apply()` function][pgroonga-wal-apply] and [the `pgroonga_vacuum()` function]"
+"[pgroonga-vacuum] automatically on stadnby database."
```

Comment:
Is there any reason to use`provides executing`?
Just using `execute` seems fine to me. 
```
"The `pgroonga_standby_maintainer` module automatically executes [the `pgroonga_wal"
"_apply()` function][pgroonga-wal-apply] and [the `pgroonga_vacuum()` function]"
"[pgroonga-vacuum] on standby database.

```